bikeshare_app/templates/checkin.html

```diff
@@ -0,0 +1,115 @@
+{% extends "layout.html" %}
+{% block title %}Check In{% endblock %}
+{% block description %}BUBS is Binghamton University's own bike share program.{% endblock %}
+
+{% block stylesheets %}
+<link rel="stylesheet" href="{{ url_for('static', filename='css/login.css') }}">
+{% endblock %}
+
+{% block navigation %}{% endblock %}
+
+{% block content %}
+<div id="registerArea" class="ui middle aligned center aligned grid">
+    <div class="column">
+        <div class="ui piled segment">
+            <a id="registerHeader" href="{{ url_for('index') }}">
+                <h2 class="ui logo header">
+                    <img src="{{ url_for('static', filename='images/logo.png') }}">
+                    <div class="content">
+                        How was your ride?
+                    </div>
+                </h2>
+            </a>
+        </div>
+        <form class="ui large form">
+            <h2 class="ui horizontal divider header">
+                <i class="check square icon"></i>
+                Does the bike have any damages?
+            </h2>
+
+            <div>
+                <div class="inline fields">
+                    <div class="field">
+                        <div class="ui radio checkbox">
+                            <input type="radio" id="damageYes" name="standing" tabindex="0" value="Yes" class="hidden">
```

Comment:
I thought the name attribute was a semantic thing because when I changed the name the radio buttons stopped working. Now I realize that that had absolutely nothing to do with it, it was just the name of the given element. I will change that. Thanks!